mflowgen/Tile_PE/construct.py

```diff
@@ -61,7 +61,7 @@ def construct():
   # Power aware setup
   if pwr_aware: 
       power_domains = Step( this_dir + '/../common/power-domains' )
-
+      #pwr_aware_gls = Step( this_dir + '/../common/pwr-aware-gls' )
```

Comment:
Why is this commented out?

Reply:
I will add power aware gls node in the next PR.